tests/components/textbox.test.jsx

```diff
@@ -41,14 +41,43 @@ describe('components/TextBox', () => {
                 value='some test text'
                 onChange={emptyFunction}
                 onKeyPress={emptyFunction}
-                characterLimit={4000}
+                characterLimit={14}
+                createMessage='placeholder text'
+                supportsCommands={false}
+                handlePostError={handlePostError}
+            />
+        );
+
+        wrapper.find('#someid').value = TestHelper.randomString('some test text that exceeds char limit');
+        expect(gotError).toEqual(true);
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should throw error when new property is too long', () => {
+        function emptyFunction() {} //eslint-disable-line no-empty-function
+
+        // this mock function should be called when the textbox value is too long
+        var gotError = false;
+        function handlePostError(msg) {
```

Comment:
Excellent -- can I get you to also add this `handlePostError` into the test case above? As it stands, it's technically not testing anything right now either because `handlePostError` is always called, but as we discovered patching this unit test, with `null` when there is no error.